package/boot/uboot-mvebu/patches/142-arm-mvebu-Espressobin-Define-fdt_addr-and-kernel_add.patch

```diff
@@ -0,0 +1,34 @@
+From fdac03df5242f80031abf84b4fa1327931381a44 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Pali=20Roh=C3=A1r?= <pali@kernel.org>
+Date: Thu, 27 Aug 2020 15:01:27 +0200
+Subject: [PATCH] arm: mvebu: Espressobin: Define fdt_addr and kernel_addr env
+ aliases
```

Comment:
That should be done in distribution boot script.

Reply:
Right, but, without those and an updated firmware your old distro might not boot anymore.
And it's reviewed by the maintainer, so most likely mainline soon:
http://patchwork.ozlabs.org/project/uboot/patch/20200827130128.22389-3-pali@kernel.org/

Reply:
Still this should be handled by boot script and can be easily backported for older branches, but well, if it's in main tree there's nothing to argue about.